### PR TITLE
Index small fixes

### PR DESCRIPTION
**Pull request by Razor421**
very small fixes

1. changed asean team page card info
2. changed my about us card info
3. and changed a very small latin fix on home page

### CODE REVIEW

File: About_Us/index.html [hunk 1]
```diff
@@ -5,24 +5,7 @@
...
-  <meta property="og:title" content="ASEAN Build The Earth">
-  <meta property="og:description" content="Official Website for ASEAN BTE">
-
-  <meta property="twitter:card" content="summary_large_image">
-  <meta property="twitter:url" content="https://asean.buildtheearth.asia">
-  <meta property="twitter:title" content="ASEAN Build The Earth">
-  <meta property="twitter:description" content="Official Website for ASEAN BTE">
-
-  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
-    integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
```
Comment (Kira272921) [inline About_Us/index.html:25]: hey dont delete the meta tags

Comment (Razor421) [reply About_Us/index.html:25]: wait i did?
when

**Review by StoneMcYT — CHANGES_REQUESTED** [review]
Stop deleting meta tags and swapping text and removing pfps.

**Comment by tintinkung** [comment]
## 🔏 Closed: `deleted and outdated pr`

**Pull request closed without merging**